base/src/main/java/org/perfmon4j/demo/DynamicStressTester.java

```diff
@@ -20,6 +20,8 @@
 */ 

```

Comment:
This class is simply a performance test used to compare two perfmon4j releases.  This is not production code.  You don't need to focus your review on this class.